### ISSUE TITLE
quotes rule false positive when string in return() statement

### ISSUE THREAD

**Issue by duaraghav8** [duaraghav8/Ethlint#240]
```
function toString(bool x) internal pure returns (string) {
    if (x) {
        return('TRUE');
    }
    return('FALSE'); 
}
```
nothing special in `.soliumrc.json`, only `"quotes": ["error", "single"]`

produces

`quotes: 'TRUE': String literal must be quoted with single quotes.`

UPDATE: also holds for `var x = ("hello world");`.

**Comment by duaraghav8** [comment]
solved in v1.2.3
https://github.com/duaraghav8/Ethlint/blob/master/CHANGELOG.md#123-2019-02-11
